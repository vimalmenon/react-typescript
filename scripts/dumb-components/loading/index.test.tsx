import "jsdom-global/register";
import * as React from "react";
import * as renderer from "react-test-renderer";
import Loading from "./index";
import {shallow} from "enzyme";

describe("Loading component :", () => {

    test("Snapshot renders :", () => {
        const component = renderer.create(<Loading />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    test("Component contains test :", () => {
        const wrapper = shallow(<Loading />);
        expect(wrapper.contains(<div id="loader-wrapper"><div id="loader"></div></div>)).toBe(true);
    });
    
});