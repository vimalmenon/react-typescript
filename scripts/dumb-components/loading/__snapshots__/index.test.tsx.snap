// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component : Snapshot renders : 1`] = `
<section
  className="sc-bdnylx fyxzQy"
>
  <div
    id="loader-wrapper"
  >
    <div
      id="loader"
    />
  </div>
</section>
`;
